EventListener/BugsnagListener.php

```diff
@@ -43,16 +51,18 @@ class BugsnagListener implements EventSubscriberInterface
     /**
      * Create a new bugsnag listener instance.
      *
-     * @param \Bugsnag\Client                                $client
-     * @param \Bugsnag\BugsnagBundle\Request\SymfonyResolver $resolver
-     * @param bool                                           $auto
+     * @param \Bugsnag\Client                                              $client
+     * @param \Bugsnag\BugsnagBundle\Request\SymfonyResolver               $resolver
+     * @param \Bugsnag\BugsnagBundle\DependencyInjection\ErrorNotifyFilter $resolver
+     * @param bool                                                         $auto
      *
      * @return void
      */
-    public function __construct(Client $client, SymfonyResolver $resolver, $auto)
+    public function __construct(Client $client, SymfonyResolver $resolver, ErrorNotifyFilter $errorNotifyFilter, $auto)
```

Comment:
Adding a new parameter here could be regarded as a breaking change.

Reply:
Indeed. It just didn't feel right to me to put it at the end. Also, as we don't provide a default value here for this field, it'll still be breaking change if put at the end.

In fact, we could even remove the `$auto` field, as we can achieve the same behavior using a `ErrorNotifyFilter` which simply returns `false` every time. A `RejectAllErrorNotifyFilter` could be provided as well.

What do you think ?